src/mainwindow.cpp

```diff
@@ -608,25 +613,21 @@ QStringList MainWindow::saveAttackParameters()
         //       http://www.openwall.com/lists/john-users/2011/08/17/2
         // We remember format key to be used with '-show' to take
         // progress.
-        m_format = "--format=" + m_ui->formatComboBox->currentText();
+        format = "--format=" + m_ui->formatComboBox->currentText();
```

Comment:
I don't like that we're also storing '--format=' along with the actual format. That should be only for the progress bar and it shouldn't include the '--' dashes.